tools/combine-files/test_files/run_tests.py

```
@@ -1,13 +1,132 @@
+        sys.executable,
+        str(script_path),
+        "--config",
+        str(config_file),
+        "--file-path-root",
+        str(test_files_dir),
+        "--output",
+        str(output_file),
+        *args,
+    ]
+
+    result = subprocess.run(cmd, capture_output=True, text=True, check=False)
```

Ruff (0.15.17)

[error] `subprocess` call: check for execution of untrusted input

(S603)